npm/src/debug-service/debug-service.ts

```diff
@@ -22,6 +22,7 @@ export interface IDebugService {
   loadSource(
     path: string,
     source: string,
+    target: string,
```

Comment:
It would be nice to strongly type this as "base" | "full" so that you force the caller to sanitize the value before calling. (See my other comment about this)